<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="UI_RenderVideo" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QWidget" name="UI_LocalVideo" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-90</y>
      <width>781</width>
      <height>681</height>
     </rect>
    </property>
    <widget class="QWidget" name="UI_LocalVideo_2" native="true">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
